test/feedcontainer.cpp

```diff
@@ -277,14 +290,25 @@ TEST_CASE("Correctly sorts feeds", "[FeedContainer]")
 		REQUIRE(sorted_feeds[3]->get_firsttag() == "tag10");
 		REQUIRE(sorted_feeds[4]->get_firsttag() == "taggy");
 	}
+}
+
+TEST_CASE("sort_feeds() sorts by feed's title if `feed-sort-order` "
+	"is \"title\"",
+	"[FeedContainer]")
+{
+	FeedContainer feedcontainer;
+	ConfigContainer cfg;
+	Cache rsscache(":memory:", &cfg);
+	const auto feeds = get_five_empty_feeds(&rsscache);
+	feedcontainer.set_feeds(feeds);
 
 	feeds[0]->set_title("tag1");
 	feeds[1]->set_title("Taggy");
 	feeds[2]->set_title("tag10");
 	feeds[3]->set_title("taggy");
 	feeds[4]->set_title("tag2");
 
-	SECTION("by title asc") {
+	SECTION("`title-asc` -- titles in dictionary order") {
```

Comment:
I think this description and the `title-desc` description still mention the wrong direction.

Reply:
And order, too! Fixed. Thank you!